tests/io/v0_0_0/test_http_backend.py

```diff
@@ -132,48 +135,6 @@ def test_numpy(self):
 
         self.assertTrue(np.array_equal(list(result.tiles())[0].numpy_array, tile.numpy_array))
 
-    def test_numpy_requires_seekable(self):
```

Comment:
If caching is allowed (which it is in this test), CachingBackend is used. CachingBackend always reads from the cache to return, and it always returns in a readable format so this test was no longer necessary, the error will never be thrown.

Reply:
Thanks!  When you merge, github might be dumb and not slurp in this text as part of your commit description.  You may have to copy and paste it over.